content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -316,11 +316,10 @@ Currently, storage size is the only resource that can be set or requested.  Futu
 
 ### Volume Mode
 
-{{< feature-state for_k8s_version="v1.13" state="beta" >}}
+{{< feature-state for_k8s_version="v1.18" state="stable" >}}
 
-Prior to Kubernetes 1.9, all volume plugins created a filesystem on the persistent volume.
-Now, you can set the value of `volumeMode` to `block` to use a raw block device, or `filesystem`
-to use a filesystem. `filesystem` is the default if the value is omitted. This is an optional API
+You can set the value of `volumeMode` to `Block` to use a raw block device, or `Filesystem`
```

Comment:
Do you want to give some context on when you would use one or the other? Also it may be good to clarify that block storage systems provide filesystem mode by handling the filesystem formatting

Reply:
Someone reading this will want to know if they can have a durable block device made available to their container and access it using, eg, a userland filesystem.
Good to make it clear what's on offer.

Reply:
Added some overview